js/ui/popup/ui.popup.js

```diff
@@ -347,6 +347,11 @@ const Popup = Overlay.inherit({
                 && parseInt(contentRect.height, 10) === this._renderedDimensions?.height;
     },
 
+    _renderContent() {
+        this.callBase();
+        this._observeContentResize(this.option('visible'));
```

Comment:
When do we unsubscribe RO? Seems like `_renderContent` is called only on showing.

Reply:
I guess we can unsubscribe [here](https://github.com/DevExpress/DevExtreme/blob/180d1d8287ef5eda0f716edd1d065fd326239a89/js/ui/popup/ui.popup.js#L429).